Makefile

```diff
@@ -90,7 +90,7 @@ build_docker:
 	docker run -w /egsrc -u ${shell id -u}:${shell id -g} --rm \
 	-v ${GO_PATH}:/gopath -v ${MKFILE_DIR}:/egsrc -v ${MKFILE_DIR}build/cache:/gocache \
 	-e GOPROXY=https://goproxy.io,direct -e GOCACHE=/gocache -e GOPATH=/gopath \
-	megaease/golang:1.17-alpine make build DOCKER=true
+	megaease/golang:1.18-alpine make build DOCKER=true
```

Comment:
This Golang image needs to be pushed to the docker hub, it is built from `build/package/Dockerfile.builder`, and this docker file should be included in this PR.

Reply:
Oh good point, it's now uploaded https://hub.docker.com/repository/docker/megaease/golang